src/presence.js

```diff
@@ -0,0 +1,35 @@
+import * as R from 'ramda';
+
+/**
+ * Check if value is empty or undefined then returns null, otherwise, returns itself.
+ *
+ * @func presence
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.29.0|v2.29.0}
+ * @category Object
+ * @sig k -> null | k
```

Comment:
Can we name the type parameter `a` and change the order of the result? It's more conventional for this codebase

```suggestion
 * @sig a -> a | null
```

Reply:
Also we could add link to `R.isEmpty` below the `@sig` tag
```js
* @see {@link http://ramdajs.com/docs/#isEmpty|R.isEmpty}
```